test/CadlRanchProjects/parameters/spread/src/Generated/tspCodeModel.json

```diff
@@ -339,7 +339,7 @@
       {
        "$id": "43",
        "Name": "SpreadAsRequestBodyRequest",
-       "NameInRequest": "SpreadAsRequestBodyRequest",
+       "NameInRequest": "",
```

Comment:
This is the side effect of working around TCGC not handling alias models. But I think it's the good change, because those are all body parameters. To them, `NameInRequest` is meaningless.